components/views/login/Login.module.scss

```diff
@@ -0,0 +1,84 @@
+.content {
+    display: grid;
+    place-items: center;
+    height: calc(100vh - var(--nav-height) - var(--footer-height));
+    .formContainer {
+        height: fit-content;
+        width: 20rem;
+        background: rgba(3, 3, 3, 0.2);
+        padding: 2rem;
+        display: flex;
+        flex-direction: column;
+        .header {
+            text-align: center;
+            color: var(--off-white);
+            padding-bottom: 1rem;
+        }
+        .errorMessage {
+            color: red;
```

Comment:
Create a theme variable for error font, and pick a hex that isnt too harshly red. replace this with that theme variable. we will be reusing this concept everywhere

Reply:
Picked out #cc0000 for the errrors. I set it to variable `--error-red`